kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/SelectDeviceForm.vue

```diff
@@ -356,6 +379,14 @@
           this.$emit('removed_address');
         });
       },
+      canLearnerSignUp(id) {
+        if (this.LODDevicesWithSignUpFacility) {
+          if (id in this.LODDevicesWithSignUpFacility) {
+            return true;
+          }
+        }
+        return false;
+      },
```

Comment:
Non blocking, but I would reduce the nesting in this function to;

```
canLearnerSignUp(id) {
    return this.LODDevicesWithSignUpFacility && id in this.LODDevicesWithSignUpFacility;
},
```
and probably use `lodsWithSignupFacility` as variable instead of  `LODDevicesWithSignUpFacility`(because LOD-Learn-only Device(s) :))